src/Main/DeathRecap.js

```diff
@@ -201,15 +201,15 @@ class DeathRecap extends React.PureComponent {
                         </td>
                         <td style={{ width: '20%' }}>
                           {event.buffsUp && event.buffsUp.sort(sortByTimelineIndex).map(e =>
-                            <SpellIcon style={{ border: '1px solid rgba(0, 0, 0, 0)' }} id={e.spell ? e.spell.id : e} />
+                            <SpellIcon style={{ border: '1px solid rgba(0, 0, 0, 0)' }} id={e.spell instanceof Array ? e.spell[0].id : e.spell.id} />
```

Comment:
Please use primarySpell instead https://github.com/WoWAnalyzer/WoWAnalyzer/blob/master/src/Parser/Core/Modules/Ability.js#L108

Reply:
Not sure why, but this doesn't work :( will try to figure out why tomorrow.